client/my-sites/checkout/checkout/wechat-payment-box.jsx

```diff
@@ -170,22 +173,30 @@ export class WechatPaymentBox extends Component {
 					/>
 
 					<div className="checkout__payment-box-actions">
```

Comment:
Fixes:
- Added `payment-box__payment-buttons`
- `pay-button` as `span`
- `button` -> `Button`
- `disabled` = pending to `disabled` = pending or failed
- Added secure payment / lock icon